configuration/rules-blockly.md

```diff
@@ -0,0 +1,157 @@
+
+This section provides a detailed description of the specific blocks and provides examples on how to use them. Note that some of the blocks (like voice, streaming or notifications) need some special setup within openHAB  - in these case links to the respective documentation is provided. It should also be mentioned that each of the blocks do have a context sensitive menu which appears upon a right click on the block itself.
+
+*Block context menu*
+
+![blockly-context.png](images/blockly/blockly-contextmenu.png)
+
+**Looking for help**
+
+A special mention should go towards the **Help** entry that links to a resource that is usually very helpful to understand the context of that particular block. Please read this information first before asking questions in the forum. *In case you ask for help please always post the respective code that is being generated.*
+
+## Before using blockly
+
+Please visit [Getting started with openHAB Blocklies](rules-blockly-before-using.html) before asking questions in the forum
+
+## Blocks
+
+### Items and Things
+
+*Items* and *Things* are the [major entities of openHAB](https://www.openhab.org/docs/concepts/) to control and monitor the home.
+
+[![Items and Things](images/blockly/blockly-items-and-things-small.png "Items and Things")
+](rules-blockly-items-things.html)
+
+See [Items & Things](rules-blockly-items-things.html) section
+
+### Timers and Delays
+
+Timers and Delays are a little more complex but add important functionality to rules. Whilst the “Wait-For”-block is straightforward, the timer blocks should be well understood before being used - they may behave differently than expected. This chapter explains what these blocks do, sometimes displaying generated code to explain what is happening behind the scenes.
+
+
+[![Timers and Delays](images/blockly/blockly-timers-and-delays-small.png "Items and Things")
+](rules-blockly-timers-and-delays.html)
+
+See [Timers and Delays](rules-blockly-timers-and-delays.html) section.
+
+### Date Handling
+
+Date blocks are used as input parameters for other blocks.
+At the moment some of these blocks are found in the ephemeris section, whilst others are found in the persistence section - they may be equally useful for both. These blocks are *typed* to assure correct connection to other blocks.
+
+[![Timers and Delays](images/blockly/blockly-date-handling-ephemeris-small.png "Ephemeris Dates")
+](rules-blockly-date-handling.html) [![Timers and Delays](images/blockly/blockly-date-handling-persistence-small.png "Persistence Dates")
+](rules-blockly-date-handling.html)
+
+See [Date Handling](rules-blockly-date-handling.html) section.
+
+### Ephemeris
+
+The ephemeris category provides blocks with calendar functionality. The blocks can be used to determine what type of day today is, or a number of days before or after today is. For example, a way to determine if today is a weekend, a bank holiday, someone’s birthday, trash day, etc.
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.